dashboard/src/pages/devices/devices.tsx

```diff
@@ -74,12 +77,26 @@ const Devices = () => {
     };
 
     const deleteOne = (id: number) => {
-        if (prompt('請輸入 delete') !== 'delete') {
+        const shouldBeDeleteDevice = (devices || []).find(
+            (item) => item.id === shouldBeDeleteId
```

Comment:
這邊有個 bug，應該要是

```suggestion
            (item) => item.id === id
```

否則永遠無法觸發